.github/workflows/build.yml

```diff
@@ -78,31 +78,74 @@ jobs:
         committer: ${{ github.actor }}
         package-names: ${{ env.DispatchPackageNames }}
 
-  dispatch-workflow:
-    if: ${{ github.repository_owner == 'GeneXusLabs' && github.base_ref == '' && (github.ref == 'refs/heads/beta' || github.ref == 'refs/heads/master') }}
+    - name: Push packages
+      if: github.repository_owner == 'GeneXusLabs' && steps.buildVariables.outputs.SHOULD_DEPLOY == 'true'
+      env:
+        GPRFeedURL: https://nuget.pkg.github.com/genexuslabs/index.json
+        NuGetFeedURL: https://api.nuget.org/v3/index.json
+      run: |
+        $IsPrerelease = [System.Convert]::ToBoolean($Env:IsPrerelease)
+        $totalPackages  = 0
+        $pushedToAzure  = 0
+        $pushedToGitHub = 0
+        $pushedToNuget  = 0
+        Get-ChildItem ".\dotnet\*.nupkg" -Recurse | ForEach-Object {
+          $PushToGitHubArgs = @("nuget", "push", $_.FullName, "--source", $Env:GPRFeedURL, "--api-key", "${{ secrets.SECURE_TOKEN }}")
+          $PushToNugetArgs  = @("nuget", "push", $_.FullName, "--source", $Env:NuGetFeedURL, "--api-key", "${{ secrets.NUGET_ORG_TOKEN }}")
+          $PushToAzureArgs  = @("nuget", "push", $_.FullName, "--source", $Env:AZURE_ARTIFACTS_URL, "--api-key", "DUMMY-KEY")
+          if ([string]::IsNullOrEmpty("${{ github.event.inputs.skip-duplicates }}") ) {
+            $skipDuplicates = $true
+          } else {
+            $skipDuplicates = [System.Convert]::ToBoolean("${{ github.event.inputs.skip-duplicates }}")
+          }
+          
+          if ($skipDuplicates) {
+            $PushToNugetArgs  += "--skip-duplicate"
+            $PushToGitHubArgs += "--skip-duplicate"
+            $PushToAzureArgs  += "--skip-duplicate"
+          }
+                    
+          dotnet $PushToAzureArgs
+          $pushedToAzure += 1
+          if (!$IsPrerelease) {
+            dotnet $PushToGitHubArgs
+            $pushedToGitHub += 1
+            dotnet $PushToNugetArgs
+            $pushedToNuget += 1
```

Comment:
Should this package be pushed to NuGet Gallery and Azure Artifacts?

Reply:
Is not Azure Artifacts the place we deploy all beta packages? Should I disable it?

Reply:
Sorry, meant to say NuGet Gallery

Reply:
Nuget is not the place we deploy Release packages?